android/src/com/appc/titanium/socketio/SocketIOClientProxy.java

```diff
@@ -0,0 +1,200 @@
+/**
+ * This file was auto-generated by the Titanium Module SDK helper for Android
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2017 by Appcelerator, Inc. All Rights Reserved.
```

Comment:
2018 or present.

Reply:
We should probably check the generator then. It should not have generated a file with 2017 as the copyright range.